notion-server/src/ethereum/mod.rs

```diff
@@ -603,9 +666,10 @@ mod tests {
         let mut eth = Simulation::new();
 
         // Add EE
-        let example_wasm_code = "some wasm code here";
+        let example_wasm_code: &[u8] = include_bytes!("../../tests/do_nothing.wasm");
```

Comment:
nice :)

Reply:
haha yes, I was quite happy coming up with this (simple) fix for having the existing test cases break as soon as the "wasm code" was actually passed to a real ewasm runtime...